Update Demonstration Scenarios with additional parameters (#53)
AI4WORK-Project/sliding-work-sharing: README.md

  --data '{
    "decisionStatus": "Sliding Decision Request",
    "slidingDecisionInputParameters": {
      "numberOfTrucksInQueue": 7,
      "positionOfTruckToBePrioritized": 5,
      "materialUrgency":30,
      "operationalWorkload":80
    }
  }'
```

To test the implemented rules, you can pass different inputs to the application and observe the outcomes. Just modify
the values for the `slidingDecisionInputParameters` as follows:

- `numberOfTrucksInQueue`: Total number of trucks in the queue (0-20 trucks)
- `positionOfTruckToBePrioritized`: Position of the truck number that needs to be prioritization (0-20)

> - `materialUrgency`: Urgency of the material that is being delivered by the truck (0%-100%)

Reviewer: ```suggestion
- `materialUrgency`: Urgency of the need for the material that is being delivered by the truck (0%-100%)
```

supervisor) should be involved in the decision if this transport should be done by the drone or by the worker. This
decision depends on:

- the drone's battery level
- the availability of the drone
- the fatigue level of the worker
- the distance from the current location of the box to the central collection point

#### Example rules

- if drone battery level is low, let supervisor decide if the drone should carry the box or not

> - if distance from the current location is low and the is drone currently available is FALSE or fatigue level of worker

Reviewer: ```suggestion
- if distance from the current location is low and the drone is currently not available or fatigue level of worker
```

  --data '{
    "decisionStatus": "Sliding Decision Request",
    "slidingDecisionInputParameters": {
      "timeTheRobotIsAlreadyMoving": 4,
      "robotBatteryStatus": 65,
      "noOfHumansInTheRoom": 10
    }
  }'
```

To test the implemented rules, you can pass different inputs to the application and observe the outcomes. Just modify
the values for the `slidingDecisionInputParameters` as follows:

> - `timeTheRobotIsAlreadyMoving`: The time that the robot is blocked, measured in minutes (0-15 minutes)

Reviewer: ```suggestion
- `timeTheRobotIsAlreadyMoving`: The time that the robot is already moving, measured in minutes (0-15 minutes); a higher time may indicate that the robot is blocked
```

- `value`: this is the original number provided as input in the Sliding Decision Request.
- `membershipValues`: this shows the "fuzzy categories" into which the input value fits. Each category is assigned a

>   membership degree between 0 and 1. In the given example, the input value of `7.0` belong to `moderate` categories with

Reviewer: ```suggestion
  membership degree between 0 and 1. In the given example, the input value of `7.0` belongs to the `moderate` category with
```
